### ISSUE TITLE
segmentation field should use either `Image` or `Labels` layers as input

### ISSUE THREAD

**Issue by quantumjot** [lowe-lab-ucl/napari-btrack#28]
Currently, in #26 only `Image` layers can be used. However, it's common to use `Labels` layers for this purpose too, which has the advantage that they can be edited directly in napari.

**Comment by alessandrofelder** [comment]
This will be very easy to fix once https://github.com/napari/magicgui/issues/93 is fixed.
Could we temporarily always expect a Labels layer until then, @quantumjot ?

**Comment by quantumjot** [comment]
> Could we temporarily always expect a Labels layer

Yes - I think that should be fine! The default should be a `Labels` layer
